storage.js

```diff
@@ -32,24 +32,24 @@ module.exports = function (generate) {
   function constructKeys(keys, legacy) {
     if(!keys) throw new Error('*must* pass in keys')
 
-    return [
-    '# this is your SECRET name.',
-    '# this name gives you magical powers.',
-    '# with it you can mark your messages so that your friends can verify',
-    '# that they really did come from you.',
-    '#',
-    '# if any one learns this name, they can use it to destroy your identity',
-    '# NEVER show this to anyone!!!',
-    '',
-    legacy ? keys.private : JSON.stringify(keys, null, 2),
-    '',
-    '# WARNING! It\'s vital that you DO NOT edit OR share your secret name',
-    '# instead, share your public name',
-    '# your public name: ' + keys.id
-    ].join('\n')
+    return `# WARNING: Do not edit, share, or use this file on two devices at once.
```

Comment:
I see that this removed the `NEVER show this to anyone!!!` with a mention of `..., share, ...`. I think some people might miss the fact that this file cannot be shown to other *people*. I mean, this warning sentence can read as "Do not ... share ... this file on two devices at once".

I prefer to shout the obvious than to be brief, so perhaps:

"Warning: do not show this file to anyone! Do not edit it. Do not copy it to other devices you own."